algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/molecular/GeneHasSufficientCopyNumberTest.kt

```diff
@@ -0,0 +1,117 @@
+package com.hartwig.actin.algo.evaluation.molecular
+
+import com.hartwig.actin.algo.evaluation.EvaluationAssert.assertMolecularEvaluation
+import com.hartwig.actin.datamodel.PatientRecord
+import com.hartwig.actin.datamodel.TestPatientFactory
+import com.hartwig.actin.datamodel.algo.EvaluationResult
+import com.hartwig.actin.datamodel.molecular.driver.GeneRole
+import com.hartwig.actin.datamodel.molecular.driver.ProteinEffect
+import com.hartwig.actin.datamodel.molecular.driver.TestCopyNumberFactory
+import com.hartwig.actin.datamodel.molecular.driver.TestTranscriptCopyNumberImpactFactory
+import com.hartwig.actin.datamodel.molecular.driver.CopyNumberType
+import org.junit.Test
+
+class GeneHasSufficientCopyNumberTest {
+    private val passingSufficientCopiesOnCanonicalTranscript = TestCopyNumberFactory.createMinimal().copy(
+        gene = "gene A",
+        geneRole = GeneRole.ONCO,
+        proteinEffect = ProteinEffect.GAIN_OF_FUNCTION,
+        isReportable = true,
+        canonicalImpact = TestTranscriptCopyNumberImpactFactory.createTranscriptCopyNumberImpact(CopyNumberType.FULL_GAIN, 40, 40)
+    )
+    private val passingSufficientCopiesOnNonCanonicalTranscript = TestCopyNumberFactory.createMinimal().copy(
```

Comment:
or:
```suggestion
    private val passingSufficientCopiesOnNonCanonicalTranscript = passingSufficientCopiesOnCanonicalTranscript.copy(
```
since most fields are the same.

Reply:
Like it, thanks :)